mobile/src/components/Button.tsx

```diff
@@ -30,7 +32,8 @@ export default function Buttons({
 
   return (
     <Pressable className={`${baseStyle} ${addButtonStyle}`}>
-      <Text className={`${textStyle} ${addTextStyle}`}>{name}</Text>
+      <Text className={textStyle}>{buttonName}</Text>
```

Comment:
If you do `buttonName &&  <Text className={textStyle}>{buttonName}</Text>` it will not render it if `buttonName` is null or undefined.